app.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+var http = require('http'), express = require('express');
+var bodyParser = require("body-parser");
+var bookApp = require('./models/bookApp');
+var app = express();
+
+
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(express.static('public'));
+
+
+app.get('/getSettings', function (req, res) {
+    res.send(bookApp.getSettings());
+});
+
+app.post('/registration', function (req, res) {
+    res.send(bookApp.registration(req.body)) ;
+});
+
+app.post('/login', function (req, res) {
+    res.send(bookApp.login(req.body)) ;
+});
+
+app.post('/getData', function (req, res) {
+    res.send(bookApp.getData(req.body)) ;
+});
+
+app.post('/postData', function (req, res) {
```

Comment:
if you have route `data` than don't name it like `getData`, `postData` - use `app.post('/data', function (req, res) { ... })`